packages/espresso/android/src/main/java/androidx/test/espresso/flutter/action/SyntheticClickAction.java

```diff
@@ -21,7 +21,7 @@
  * <p>Note, this is not a real click gesture event issued from Android system. Espresso delegates to
  * Flutter engine to perform the {@link SyntheticClick} action.
  */
-@Beta
+@ExperimentalTestApi
```

Comment:
Will this change break our tests in other packages, given that it has `@RequiresOptIn`?

Reply:
I think the best option here since we are changing public API annotations in way that requires client action is to bump the version to `0.3.0` for this change.

Reply:
Done